lib/src/slices/api_microsoft/api_microsoft_service_email.dart

```diff
@@ -0,0 +1,57 @@
+import '../api_auth_service/api_auth_service.dart';
+import '../api_auth_service/model/api_auth_service_account_model.dart';
+import '../api_email_msg/model/api_email_msg_model.dart';
+import '../data_bkg/data_bkg_sv_email_prov.dart';
+import '../data_bkg/model/data_bkg_model_page.dart';
+import 'api_microsoft_service.dart';
+
+class ApiMicrosoftServiceEmail extends ApiMicrosoftService
```

Comment:
Should not extend the base service. The base service should contain a list of services it supports

Reply:
TBD... just to remind me of this when I get back